app/views/complaints/_tta_activity_report_form.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="ct-section-divider">
+  <p class="display-flex flex-justify">
+    <span> Activity Report ID</span>
+    <span><%= link_to "Cancel", "#js-add-tta-report", id: "js-close-tta-form", role: "button" %></span>
+    <div>
+      <%= form_with url: issue_tta_reports_path, method: "post", class: "usa-search display-flex", id: "tta-report-form"  do %>
+        <%= label_tag "tta-report-id", "TTA report display id", id: "tta-report-label" %>
```

Comment:
[suggestion] Update the label to `TTA report display ID` so that VoiceOver pronounces each of the letters of I.D. instead of the end of `did`